CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # 0.3.0 - Unreleased
 
+* Compeletly rebuilt database driver to use the [KnexJS](https://knexjs.org/) query builder. #174
+    * **Upgrades of existing SQLite databases must be performed from v0.2.3**
+    * Add's support for **MySQL**.
```

Comment:
```suggestion
    * Adds support for **MySQL**.
```

Reply:
I need to stop doing this stuff in the middle of the night lol